sql/15.0/sqlite_fdw.sql

```diff
@@ -656,9 +660,1001 @@ DROP FOREIGN TABLE grem1_1;
 DROP FOREIGN TABLE grem1_2;
 --Testcase 235:
 DROP FOREIGN TABLE case_exp;
+--test for PR #76 github
+DROP FOREIGN TABLE "Unicode data";
 
 --Testcase 151:
 DROP SERVER sqlite_svr;
 --Testcase 152:
 DROP EXTENSION sqlite_fdw CASCADE;
 
+-- tests for PR #76 github
+-- euc_jp
+CREATE DATABASE "contrib_regression_EUC_JP" ENCODING EUC_JP LC_CTYPE='ja_JP.eucjp' LC_COLLATE='ja_JP.eucjp' template template0;
+\connect "contrib_regression_EUC_JP"
+CREATE EXTENSION sqlite_fdw;
+CREATE SERVER sqlite_svr FOREIGN DATA WRAPPER sqlite_fdw
+OPTIONS (database '/tmp/sqlitefdw_test.db');
+CREATE FOREIGN TABLE "Unicode data"(i text OPTIONS (key 'true'), t text) SERVER sqlite_svr; 
+-- EUC_JP
+SELECT * FROM "Unicode data" WHERE i = 'jap';
+SELECT * FROM "Unicode data" WHERE t LIKE 'いろはにほ%';
+INSERT INTO "Unicode data" (i, t) VALUES ('jap+', 'い');
```

Comment:
Could you please insert more character for each encoding (instead of 1-2 characters as current)
Example
INSERT INTO "Unicode data" (i, t) VALUES ('jap+', 'い'); --> INSERT INTO "Unicode data" (i, t) VALUES ('jap+', '今日は良い日です');

Reply:
@bichht0608, you are thinking like in ICU development group. I have used 1-3 encoding specific characters. I think we have a big test case overhead for simple `if` around `pg_do_encoding_conversion`, because not only sqlite_fdw context is tested, also we try to write outer test in addition to PostgreSQL `pg_do_encoding_conversion` test.
No problem to expand examples to full sentences with some equal final character to have difference with first sample data.